tests/spec/transformers/valid/RewriteAmpUrls/adds_esm/input.html

```diff
@@ -1,3 +1,8 @@
+<!--
+{
+  "esmModulesEnabled": true
```

Comment:
If `esmModulesEnabled`, then why aren't aren't the `mjs` scripts and `nomodule` showing up here? May be same thing as I saw in https://github.com/ampproject/amp-wp/pull/6037#pullrequestreview-627637047.

Reply:
This is the input file, so the non-optimized version. The output file does contain the expected files: https://github.com/ampproject/amp-toolbox-php/blob/main/tests/spec/transformers/valid/RewriteAmpUrls/adds_esm/expected_output.html